### PR TITLE
KOGITO-779: [DMN] Kogito webapp - Cleans

### PR DESCRIPTION
**Pull request by romartin**
Hey @manstis @tiagobento 

Although maybe has no impact on plugins, and I've this stuff recent/"fresh", I also did same cleans as done for the BPMN kogito webapp in https://github.com/kiegroup/kie-wb-common/pull/3203

It updates the DMN kogito webapp as:
* Reduced the WAR from 65Mb to 8Mb ( just by dropping all `WEB-INF/lib` ? ) - Declareted ALL  dependencies to `provided` scope - If I properly understand, we should be "pure" client side
* Add the `test.html` page as default, which provides some helper buttons to run the war in standalone mode (as bpmn)
* Removed references to Wildfly stuff 
* Added some `README.md` for quick usage
* Some other cleans..

As I've done for the BPMN kogito webapp, I tested:
* Deploying the war in an app. server (Wildfly)
* Runnning SDM via comand line
* Running SDM via IntelliJ
* The VSCode plugin generated afer this

Would be great if you can guys review this. My goal for both BPMN/DMN kogito webapps is to reduce the WAR artifact, and other "old" stuff related to the webapp, to the minimum, so it's easier to maintain and consume. 

So if these changes here are looking good to the people, it will help to also verify https://github.com/kiegroup/kie-wb-common/pull/3203

Assembly with:
https://github.com/kiegroup/kie-wb-common/pull/3203
https://github.com/kiegroup/kogito-tooling/pull/71

Thanks for your help!

### CODE REVIEW

**Comment by romartin** [comment]
Jenkins execute full downstream build

File: kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+Kogito DMN Editor - Webapp
+==============================
+
+Building
+--------
+ 
+* Before building this webapp, you should build whole the Stunner project before
```
Comment (manstis) [inline kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/README.md]: Reword ("Before building this webapp, you should build _the whole of_ the Stunner project before").

File: kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/src/main/webapp/test.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+<div id="editor">
+    Wait until the editor is loaded to open a DMN file:<br>
+    <input type="button" onclick="loadDiagram('')" value="Create new Diagram"/>
+    or
+    <input type="file" onchange="openFile(event)" value="Choose DMN file" accept="dmn,txt"/>
+    When you are ready you can
+    <input type="button" id="download_btn" value="Download the diagram" onclick="download()" disabled/>
+</div>
+
+<iframe id="editorFrame" src="index.html" width="100%" height="1000px" frameBorder="0" scrolling="no"></iframe>
```
Comment (manstis) [inline kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/src/main/webapp/test.html]: I prefer `880px` (as it does not add a vertical scrollbar for me!)

File: kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/README.md (hunk 1 above)
Comment (romartin) [reply kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/README.md]: great, good to know, thanks @manstis !

File: kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/src/main/webapp/test.html (hunk 2 above)
Comment (romartin) [reply kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/src/main/webapp/test.html]: great point too

**Comment by romartin** [comment]
Hey @manstis 

Thanks for pointing out, good points! this way I also created: https://github.com/kiegroup/kogito-tooling/pull/71

**Comment by romartin** [comment]
Jenkins execute full downstream build

File: kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
+Kogito DMN Editor - Webapp
```
Comment (jomarko) [inline kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/README.md]: can we please also update the `kie-wb-common-dmn/README.MD` to not duplicate information and point to this new README in the appropriate section?

Comment (romartin) [reply kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/README.md]: done - added a reference to this README from the root one

**Comment by romartin** [comment]
Hey @jomarko 

We still need those dependencies for the `build` and `test` classpath, for example to make the gwt complier run properly, or for running unit test. But after project building, we don't need those exported in the WAR, as we already generated all client side stuff at compile time.

So that's why, at least for now, I declared the dependencies as `provided`. We can do some more iterations on the dependencies being declared for the kogito webapp's, and perform some more cleans or maybe even remove some ones, but that still requires time we don't want to spent.

Hope it clarifies! :)

Please approve if you're happy! :)

**Comment by romartin** [comment]
Jenkins retest this

**Comment by romartin** [comment]
Jenkins execute full downstream build

**Comment by ederign** [comment]
Jenkins execute full downstream build

**Comment by romartin** [comment]
Jenkins retest this

**Comment by romartin** [comment]
Jenkins execute full downstream build

**Pull request merged**